.travis.sh

```diff
@@ -86,6 +86,8 @@ case "$JDK" in
   ;;
 9-ea | 9-ea-stable)
   # see https://bugs.openjdk.java.net/browse/JDK-8131041 about "java.locale.providers"
```

Comment:
@marchof you mean change order of comments? Unfortunately it can't be placed exactly above line or on line to which it refers.